docs/content/v2024.2/explore/observability/active-session-history.md

```diff
@@ -83,6 +83,31 @@ This view provides a list of wait events and their metadata. The columns of the
 | pid | bigint | PID of the process that is executing the query. For YCQL and background activites, this will be the YB-TServer PID. |
 | client_node_ip | text | IP address of the client which sent the query to YSQL/YCQL. Null for background activities. |
 | sample_weight | float | If in any sampling interval there are too many events, YugabyteDB only collects `ysql_yb_ash_sample_size` samples/events. Based on how many were sampled, weights are assigned to the collected events. <br><br>For example, if there are 200 events, but only 100 events are collected, each of the collected samples will have a weight of (200 / 100) = 2.0 |
+| ysql_dbid | oid | Database OID of the YSQL database. This is 0 for YCQL databases.  |
+
+### yb_wait_event_desc
+
+This view displays the class, type, name, and description of each wait event. The columns of the view are described in the following table.
+
+| Column | Type | Description |
+| :----- | :--- | :---------- |
+| wait_event_class | text | Class of the wait event, such as TabletWait, RocksDB, and so on. |
+| wait_event_type | text | Type of the wait event such as CPU, WaitOnCondition, Network, Disk IO, and so on. |
+| wait_event | text | Name of the wait event. |
+| wait_event_description | text | Description of the wait event. |
+
+## Constant query identifiers
+
+These fixed constants are used to identify various YugabyteDB background activities. They also include a placeholder "query id" for instances where the actual "query id" has not been calculated yet, but the ASH collector samples the system. The query ids of the fixed constants are described in the following table.
```

Comment:
@aishwarya24 @abhinab-yb  The second part of statement is  not needed. 
`They also include a placeholder "query id" for instances where the actual "query id" has not been calculated yet, but the ASH collector samples the system.`   ------>    `They also include a placeholder "query id" for instances where the actual "query id" has not been calculated yet.`